acceptance/steps/helm.go

```diff
@@ -0,0 +1,23 @@
+package steps
+
+import (
+	"context"
+
+	"github.com/cucumber/godog"
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/yaml"
+
+	framework "github.com/redpanda-data/redpanda-operator/harpoon"
+	"github.com/redpanda-data/redpanda-operator/pkg/helm"
+)
+
+func iInstallHelmRelease(ctx context.Context, t framework.TestingT, name string, values *godog.DocString) {
+	var valuesMap map[string]any
+	require.NoError(t, yaml.Unmarshal([]byte(values.Content), &valuesMap))
+
+	t.InstallLocalHelmChart(ctx, "../charts/redpanda", helm.InstallOptions{
+		Name:      name,
+		Namespace: t.Namespace(),
+		Values:    valuesMap,
+	}, helm.Dependency{Name: "redpanda", Repository: "https://charts.redpanda.com"})
```

Comment:
Why does redpanda depend on redpanda? 🤔

Reply:
It's interesting how we define dependency. It's more of a helm repository url that is a dependency to build Redpanda helm chart dependency which is console. 

I think better name would be Dependant helm repository address.